Common/src/CConfig.cpp

```diff
@@ -3022,6 +3023,11 @@ void CConfig::SetConfig_Parsing(istream& config_buffer){
             newString.append("DYNAMIC_ANALYSIS is deprecated. Use TIME_DOMAIN instead.\n\n");
           else if (!option_name.compare("SPECIES_USE_STRONG_BC"))
             newString.append("SPECIES_USE_STRONG_BC is deprecated. Use MARKER_SPECIES_STRONG_BC= (marker1, ...) instead.\n\n");
+          else if (!option_name.compare("INC_DENSITY_INIT"))
+            newString.append(
+                "INC_DENSITY_INIT is deprecated. The initial density is computed using the ideal gas law based on the "
+                "THERMODYNAMICS_PRESSURE and INC_TEMPERATURE_INIT. For Using the FLUID_MODEL=CONSTANT_DENSITY, the "
+                "constant density must be given using the following option DENSITY_CONSTANT.\n\n");
```

Comment:
We just released a major version, we are not breaking backward compatibility for an important setting like this.

Reply:
Sure, we can wait for a major version to implement a change in config option. But now, sometimes we use thermodynamic pressure and sometimes we use density (to compute thermodynamic pressure) and I think the inconsistency should be fixed at some point, and I think it should be fixed in this way.

Reply:
I'm not saying we cannot fix things, but the option needs to be deprecated more gracefully than this.
As for the inconsistency between density init and pressure init we can use a similar strategy to the compressible solver.
```
/*!
 * \brief Types of initialization option
 */
enum class FREESTREAM_OPTION {
  TEMPERATURE_FS, /*!< \brief Temperature initialization. */
  DENSITY_FS, /*!< \brief Density initalization. */
};
```